### PR TITLE
? FE/S3 - Implement file uploading mechanics [Q]

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/33z1mch

### CODE REVIEW

File: api/src/features/assets/asset.service.ts [hunk 1]
```diff
@@ -3,17 +3,60 @@ import { InjectRepository } from '@nestjs/typeorm';
...
+
+  createAsset(asset: AssetCreateResponse) {
+    return this.assetsRepository.save(asset);
+  }
+  async dataHandler(assetAttr: AssetCreateResponse) {
+    const imagesUrlArray = await this.uploadPublicFiles(
+      assetAttr.images,
+      assetAttr.scAddress,
+    );
+    console.log('imagesUrlArray', imagesUrlArray);
```
Comment (ruslanpalagin) [inline api/src/features/assets/asset.service.ts:49]: rm log please

File: frontend/app/features/assets/controllers/NewAssetController.tsx [hunk 2]
```diff
@@ -55,7 +57,46 @@ export const NewAssetController = () => {
...
+      averageCoCPercentage: investmentReturnForm.values.coc,
+      projectedAssetValueAppreciation: investmentReturnForm.values.projectedAssetValueAppreciation,
+      annualGrossRent: rentalManagementForm.values.annualGrossRent,
+      taxesPercentage: rentalManagementForm.values.taxes,
+      insurancePercentage: rentalManagementForm.values.insurance,
+      propertyManagementPercentage: rentalManagementForm.values.propertyManagement,
+      utilitiesPercentage: rentalManagementForm.values.utilities,
+      llcAdministrationFeePercentage: rentalManagementForm.values.llcAdministrationFee,
+    }
+    axios.post('http://localhost:3001/assets/create', {
```
Comment (ruslanpalagin) [inline frontend/app/features/assets/controllers/NewAssetController.tsx:90]: use core/http-client

File: frontend/package.json [hunk 3]
```diff
@@ -25,6 +25,7 @@
     "@types/react-dom": "18.0.9",
     "@types/react-select-country-list": "^2.2.1",
     "@typescript-eslint/eslint-plugin": "^5.45.0",
+    "axios": "^1.2.1",
```
Comment (ruslanpalagin) [inline frontend/package.json:28]: no need for axios. use core/http-client

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

File: api/src/features/assets/asset.service.ts (hunk 1 above)
Comment (Space-zee) [reply api/src/features/assets/asset.service.ts:49]: +

**Review by ruslanpalagin — APPROVED** [review]

**Pull request merged**